src/sentry/conf/server.py

```diff
@@ -1576,10 +1576,10 @@ def create_partitioned_queues(name):
         "environment": {"POSTGRES_DB": "sentry", "POSTGRES_HOST_AUTH_METHOD": "trust"},
         "volumes": {"postgres": {"bind": "/var/lib/postgresql/data"}},
         "healthcheck": {
-            "test": ["CMD", "pg_isready"],
-            "interval": 1000000000,  # Test every 1 second (in ns).
-            "timeout": 1000000000,  # Time we should expect the test to take.
-            "retries": 5,
+            "test": ["CMD-SHELL", "pg_isready -U postgres"],
```

Comment:
IDK why, but using `CMD` in combination with specifying the username (`-U postgres`) fails the health check.

`CMD-SHELL` requires that an image is shipped with a shell, so this shouldn't affect us.

Reply:
Ooh that makes sense, u da best josh.

Reply:
thx billy